schemas/event.yml

```diff
@@ -276,3 +276,11 @@
 
         This is mainly useful if you use more than one system that assigns
         risk scores, and you want to see a normalized value across all systems.
+
+    - name: ingested
+      level: core
+      type: date
+      short: Ingest timestamp
+      description: >
+        Time when the event was ingested. This is different from `@timestamp`
+        which is when the event originally occurred.
```

Comment:
I think we should be a little more explicit in the relationship with `event.created` as well here.

Here's what I'm thinking:

```YAML
      short: Timestamp when an event arrived in the central data store
      description: >
        Timestamp when an event arrived in the central data store.

        This is different from `@timestamp`, which is when the event originally occurred.
        It's also different from `event.created`, which is meant to capture the first time an agent saw the event.

        In normal conditions, assuming no tampering, the timestamps should chronologically look like this: `@timestamp` < `event.created` < `event.ingested`.
```

Just like `@timestamp` and `event.created`, the description is pretty verbose in order to clarify the relationship. I think we should fast-track this PR and we may not necessarily need to adjust `@timestamp` and `event.created` descriptions for now.